### ISSUE TITLE
Shortcut/Quick actions menu

### ISSUE THREAD

**Issue by SpeeterYT** [Moustachauve/WLED-Native-Android#21]
It would be great to have a shortcut for turning x on, x off, toggle x, turn all on, toggle all etc... (very specific, but "If all lights have the same state -> toggle, else turn all off would be amazing)
This would be very helpful for creating NFC tags which I can use to control the lights

**Comment by Moustachauve** [comment]
I think you could create NFC tags that contains the URL to the WLED JSON Api to do that?

**Comment by Snapyou2** [comment]
Hi, thanks for the project!
I was wondering if some buttons in the quick access menu (android tiles above the notification area) could be added for the app? A simple "on/off" button would already be quite useful as QoL improvement (to switch all the connected lights at once).

**Comment by Moustachauve** [comment]
@SpeeterYT said in the duplicated issue:

> Not really, for that I would need to know and manually add all ips

Ah, so you want actions that could apply to a list of devices or all devices?

**Comment by Moustachauve** [comment]
@Snapyou2 Yes! this is planned :)

**Comment by SpeeterYT** [comment]
> [@SpeeterYT](https://github.com/SpeeterYT) said in the duplicated issue:
> 
> > Not really, for that I would need to know and manually add all ips
> 
> Ah, so you want actions that could apply to a list of devices or all devices?

Exactly + even if you knew and hardcoded the IPs, you would still need some app to actually send the request (like nfctools/tasker etc.). A simple "turn {device id}/all on, off,  toggle" uri would be enough. If you add groups in the app, you could also have these uris for groups

**Comment by mhlsf** [comment]
Hi ! I would like to help and implement the feature to turn on/off all the light as I needed it also ! Is the development already started on your end ? If not, what implementation were you thinking UI wise ?

**Comment by Moustachauve** [comment]
Btw a lot of this could be implemented easily with apps like Home Assistant

**Comment by SpeeterYT** [comment]
> Btw a lot of this could be implemented easily with apps like Home Assistant

But for that you need a running HA server...

**Comment by Moustachauve** [comment]
Of course! I plan to support this in the future. The same system would be able to be used by widgets too (add quick action buttons on the widgets).
